directord/client.py

```diff
@@ -179,7 +179,6 @@ def _job_executor(
 
         self.log.debug("Running component:%s", command.decode())
         component_kwargs = dict(cache=None, job=job)
-        conn.start_processing()
```

Comment:
I think this still needs to be there or it doesn't send. nothing else appears to call this